websites/api-docs-smoke-test/src/api-specs/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.amfgraph.raml
```

Comment:
is it possible to just not include the files in the merge?

Reply:
I think it's not a good idea to rely on all team members to just not forget to commit, especially with so much automation and CI in place. 
Our  .gitignores  often contain a broad set of ignores that cover all potential developer situations to avoid accidental commits.  

if we remove the script though, then the ignore has to go, too - that's right.

Reply:
Yes, just removing the scripts works too and we won't need to ignore them :)